index.html

```diff
@@ -4,35 +4,97 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>MyStayl</title>
   <link rel="stylesheet" href="my.css">
+  <script src="app.js"></script>
+  <title>document</title>
 </head>
- <style>
-Ul {
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-} 
-</style>
-</head>
-<body> 
-  <div class="qqqq">
-    <a class="wwwww" href="default.asp">Home</a>
-    <a href="news.asp">News</a>
-    <a href="contact.asp">Contact</a>
-    <a href="about.asp">About</a>
-  </div>
-  <div class="eeee">
-    <a class="rrrr" href="default.asp">Home</a>
-    <a href="news.asp">News</a>
-    <a href="contact.asp">Contact</a>
-    <a class="tttt" href="about.asp">About</a>
+<body>
+    <header class="header">
+      <div class='container'>
+        <div class="header__body">
+          <a href="#" class="header__logo">
+            <img src="logo.jpg" alt="LogoOnHeader">
+          </a>
+          <div class="header__burger" onclick="switchClass()">
+            <div class="header__burger__middle"></div>

```

Comment:
Actually, you don't need this div element.

Reply:
I don't know how else to draw a stick in the center of a burger